### PR TITLE
address comments from CRAN

### PR DESCRIPTION
**Pull request by LittleBeannie**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -15,7 +15,16 @@ Authors@R: c(
     person("Merck Sharp & Dohme Corp", role = c("cph"))
     )
 Description: A metadata structure for clinical data analysis
-    and reporting based on Analysis Data Model (ADaM) datasets.
+    and reporting based on Analysis Data Model (ADaM) datasets.
```
Comment (fb-elong) [inline DESCRIPTION:18]: A metadata structure for clinical data analysis
    and reporting based on Analysis Data Model (ADaM) datasets. 
     The package simplify clinical analysis and reporting tool development
 by defining standardized input, output and workflow. For example, 
    the package can be used to create analysis and reporting planning grid; 
    mock table, and validated A&R results based on consistent input.

File: R/outdata.R [hunk 2]
```diff
@@ -43,7 +43,7 @@
 #' @export
 #'
 #' @examples
-#' metalite:::outdata(
+#' outdata(
```
Comment (fb-elong) [inline R/outdata.R:46]: @nanxstats, would you feel we need a global search of `:::` and remove them. 

We had examples for internal function that use `:::`. For example 

https://github.com/Merck/metalite/blob/237da7f6219c5dd8db889bed159b857309a260ce/R/meta_check.R#L31

Comment (fb-elong) [reply R/outdata.R:46]: ah, maybe they are OK, as I saw `@noRd` in those internal functions. Maybe we just need to check if there is `:::` in the `man` folder. :)

Comment (LittleBeannie) [reply R/outdata.R:46]: I just checked all Rd files and found `:::` does not appear.

File: DESCRIPTION [hunk 3]
```diff
@@ -16,6 +16,10 @@ Authors@R: c(
     )
 Description: A metadata structure for clinical data analysis
     and reporting based on Analysis Data Model (ADaM) datasets.
+    The package simplify clinical analysis and reporting tool development
```
Comment (nanxstats) [inline DESCRIPTION:19]: We should avoid obvious grammatical mistakes...

**Review by nanxstats — CHANGES_REQUESTED** [review]

**Review by nanxstats — APPROVED** [review]
I made a few minor changes, now looks good!

You do need to go through the release check list: https://github.com/Merck/metalite/issues/125

**Comment by LittleBeannie** [comment]
> I made a few minor changes, now looks good!
> 
> You do need to go through the release check list: #125

These are excellent changes. Thanks!

**Pull request merged**